lib/content-services/src/lib/search/components/search-date-range-advanced-tabbed/search-date-range-advanced-tabbed.component.spec.ts

```diff
@@ -0,0 +1,181 @@
+    @Output()
+    changed = new EventEmitter<Partial<SearchDateRangeAdvanced>>();
+    @Output()
+    valid = new EventEmitter<boolean>();
+}
+describe('SearchDateRangeAdvancedTabbedComponent', () => {
+    let component: SearchDateRangeAdvancedTabbedComponent;
+    let fixture: ComponentFixture<SearchDateRangeAdvancedTabbedComponent>;
+
+    async function clickApplyBtn() {
+        const applyBtn: HTMLButtonElement = fixture.debugElement.query(By.css('[data-automation-id="date-range-advanced-btn-apply"]')).nativeElement;
+        applyBtn.click();
+        fixture.detectChanges();
+        await fixture.whenStable();
+    }
+
+    async function clickResetBtn() {
+        const clearBtn: HTMLButtonElement = fixture.debugElement.query(By.css('[data-automation-id="date-range-advanced-btn-clear"]')).nativeElement;
+        clearBtn.click();
+        fixture.detectChanges();
+        await fixture.whenStable();
+    }
+
+    beforeEach(async () => {
+        await TestBed.configureTestingModule({
+            declarations: [SearchDateRangeAdvancedTabbedComponent, SearchFilterTabbedComponent, SearchDateRangeAdvancedComponent],
+            imports: [
+                TranslateModule.forRoot(),
+                ContentTestingModule
+            ],
+            providers: [
+                { provide: SearchFilterTabbedComponent, useClass: MockSearchFilterTabbedComponent },
+                { provide: SearchDateRangeAdvancedComponent, useClass: MockSearchDateRangeAdvancedComponent }
+            ]
+        });
+        fixture = TestBed.createComponent(SearchDateRangeAdvancedTabbedComponent);
+        component = fixture.componentInstance;
+        component.id = 'dateRangeAdvanced';
+        component.context = {
+            queryFragments: {
+                dateRangeAdvanced: ''
+            },
+            update: jasmine.createSpy('update')
+        } as any;
+        component.settings = {
+            hideDefaultAction: false,
+            dateFormat: 'dd-MMM-yy',
+            maxDate: 'oday',
+            field: 'test-field-1, test-field-2',
+            displayedLabelsByField: {
+                'test-field-1': 'Test Field 1',
+                'test-field-2': 'Test Field 2'
+            }
+        };
+        component.tabsValidity = {
+            'test-field-1': true,
+            'test-field-2': true
+        };
+        fixture.detectChanges();
+    });
+
+    it('should update displayValue when values are submitted', async () => {
+        spyOn(component.displayValue$, 'next');
+        component.combinedValuesToDisplay = 'test-combined-values-to-display';
+        await clickApplyBtn();
+        expect(component.displayValue$.next).toHaveBeenCalledWith('test-combined-values-to-display');
+    });
+
+    it('should clear values when widget is reset', async () => {
+        spyOn(component.displayValue$, 'next');
+        component.combinedValuesToDisplay = 'test-combined-values-to-display';
+        await clickResetBtn();
```

Comment:
You can remove await for clickResetBtn when you apply my changes for clickResetBtn